site/styles/main.scss

```diff
@@ -81,3 +81,7 @@ a {
       color: $color-accent;
    }
 }
+
+.hide {
```

Comment:
There is already a basic style for this -- check reset.css -- but if there weren't it should go in the component styles :)